README.md

```diff
@@ -423,6 +423,24 @@ end
 <%= component "button", label: "Sign in", url: sign_in_path %>
 ```
 
+### Validation
+
+To ensure your components get initialized properly you can use ActiveModel::Validations in your elements or components:
+
+```ruby
+# app/components/button_component.rb %>
+
+class ButtonComponent < Components::Component
```

Comment:
And if we move this section up, perhaps use the `AlertComponent` for the example instead.

Reply:
np